src/Phinx/Db/Adapter/SQLiteAdapter.php

```diff
@@ -798,7 +814,7 @@ protected function calculateNewTableColumns($tableName, $columnName, $newColumnN
         $selectColumns = array_map([$this, 'quoteColumnName'], $selectColumns);
         $writeColumns = array_map([$this, 'quoteColumnName'], $writeColumns);
 
-        if (!$found) {
+        if (!$found && $columnName !== false) {
```

Comment:
Please don't force push. It is not needed, and just reverted my fix.

Reply:
if (!$found && $columnName) {

Reply:
Done.

I've been force-pushing, so as to make the history a bit cleaner as I've had to look back a handful of times on commits and getting pieces of a larger changeset has been tedious for me.

I will stop combining my PR commits though going forward though if so desired.